### DISCUSSION TITLE
Support other frameworks

### DISCUSSION THREAD

**Discussion by 93578237** [snok/asgi-correlation-id#16] [Ideas]
Hi. Would you mind supporting other frameworks, not `starlette`-based, like [BlackSheep](https://github.com/Neoteroi/BlackSheep)?
Maybe use `functools.singledispatch` on entrypoint function by checking `app` type? At least there will be different `__call__` implementations.

**Comment by sondrelg** [comment]
Sorry I must have missed this. I don't see why not :+1: What changes would it require?
